packages/govern-core/contracts/OptimisticQueue.sol

```diff
@@ -13,6 +13,8 @@ import "./lib/MiniACL.sol";
 import "./lib/OptimisticQueueStateLib.sol";
 import "./lib/SafeERC20.sol";
 
+
+// Note: Not super fan of this name tbh
```

Comment:
me neither, i like optimistic but not queue. suggestions @sohkai @lkngtn @john-light?